src/gui/screen_menu_esp_update.cpp

```diff
@@ -212,6 +212,7 @@ void ScreenMenuESPUpdate::windowEvent(EventLock /*has private ctor*/, window_t *
         }
         case esp_upload_action::Reset:
             _dbg("ESP finished flahing");
+            help.SetText(_("ESP succesfully flash, WiFI initiation started."));
```

Comment:
Past tense (`flashed`)?

Reply:
this is visible only for debugger, but no problem